src/Juvix/Core/Common/Context.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE DeriveTraversable #-}
```

Comment:
Are we using this extension?

Reply:
to get the following deriving
```haskell
  deriving (Show, Traversable)
```

It seems we can't derive Traversable via `via` for whatever reason.

If we ever do something with efffects on this structure, then we'll need traverse

Reply:
Ah yes, misread.